packages/filesystem/src/browser/file-service.ts

```diff
@@ -394,6 +461,13 @@ export class FileService {
      */
     readonly onDidRunOperation = this.onDidRunOperationEmitter.event;
 
+    /**
+     * Try to resolve file information and metadata for the given resource.
+     * @param resource `URI` of the resource that should be resolved.
+     * @param options  Options to customize the resolvment process.
```

Comment:
```suggestion
     * @param options  Options to customize the resolved process.
```

If `resolvement` is the word we want here then I believe this should be spelling.

Reply:
Should be `resolvement`